src/pybind/mgr/cephadm/services/osd.py

```diff
@@ -79,9 +79,10 @@ async def all_hosts() -> List[Optional[str]]:
 
     async def create_single_host(self,
                                  drive_group: DriveGroupSpec,
-                                 host: str, cmd: str, replace_osd_ids: List[str],
+                                 host: str, cmds: List[str], replace_osd_ids: List[str],
                                  env_vars: Optional[List[str]] = None) -> str:
-        out, err, code = await self._run_ceph_volume_command(host, cmd, env_vars=env_vars)
+        for cmd in cmds:
+            out, err, code = await self._run_ceph_volume_command(host, cmd, env_vars=env_vars)
```

Comment:
should we check for each `run_ceph_volume_command` to make sure if it has failed or not? .. the following check `code == 1` only takes in account the last command.

Reply:
yeah, the following block checking the code should be part of the for loop. I'll change that.